### PR TITLE
Addressing requested edits

### PR DESCRIPTION
**Pull request by brandonhundt**
https://docs.google.com/document/d/1MA8rP-J5nZS7l8BFNoZdwLEua9TMSy1RDiY-T7RLcuw/edit

Highlights:
- Opens links in external windows
- images for donor quotes (but no arrows)
- Creates separate page for Planned Giving
- Moves the training page into a new path.
- A lot of image slides and image changes

## Review Steps
- [ ] Ensure you can scroll through the entirety of the report

### CODE REVIEW

File: _includes/cards/audio.html [hunk 1]
```diff
@@ -1,9 +1,30 @@
 <figure class="mt-4">
-  <header class="h4">Audio Testimonial</header>
+  {%- if page.audio-quote -%}
+  
+  <blockquote class="quote row">
+    <div class="col-md-2 quote-text quote-bg d-flex align-items-center justify-content-center">
+      <audio controls src="{{ page.audio }}" preload="none">
+        <a href="{{ page.audio }}"> Download audio </a>
+      </audio>
```
Comment (brandonhundt) [inline _includes/cards/audio.html:12]: @rpeterman-gp This is where I brought in the quote styling to the audio include...

**Comment by brandonhundt** [comment]
FYI @indraprx is taking this PR over to add alt text to images and double check the content.

**Review by rpeterman-gp — APPROVED** [review]
LGTM. I add the play button for the audio quote and fixed a bug with page inks that was breaking navigation flow. :shipit:  if you like the button.

**Comment by brandonhundt** [comment]
The audio play button is dope. Thanks @rpeterman-gp

**Pull request merged**